js/articleController.js

```diff
@@ -0,0 +1,24 @@
+articleController = {};
+
+articleController.initIndex = function() {
+  $.ajax({
```

Comment:
Much of the other data-grabbing behavior is already on the model (article.js). You could move much of this there as well, if you like.